spec/features/lateness_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
```

Comment:
"It tests that you end up back on the homepage after doing whatever it was the individual features say"

Fair enough, although if there's no way to make it *not* do that, then are we really testing our app.

I think you're right that the lateness task is a bit odd, but couldn't we add something such that if you answer "Yes, I am in time" and then you try to add a reason, it raises an error (or just that the "reason" field is not shown)?